includes/class-reader-activation.php

```diff
@@ -386,6 +386,11 @@ private static function get_element_class_name( ...$parts ) {
 	 * Setup nav menu hooks.
 	 */
 	public static function setup_nav_menu() {
+		$account_page = \wc_get_page_id( 'myaccount' );
+		if ( get_the_ID() === $account_page ) {
+			return;
+		}
```

Comment:
I feel like this creates a UI inconsistency, almost like it's glitchy. Why would we hide the account link while on the page?

Reply:
I thought there would be little point in being able to open the login modal on the standalone login page. Additionally, doing so would require refactoring the JS a bit more to be able to handle multiple instances of the login form on the same page—not a big deal, but is it worth it for this feature?

Reply:
Fair point!

Reply:
You know, I rethought this and I think you were right. 9db42f5 restores the modal on My Account pages and does the JS refactoring handle multiple instances of the form on the same page.